doc/source/changelog.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 ^^^^^^^^^
 
+Pantab 3.0.3 (2023-12-18)
```

Comment:
Minor issue with putting a date - readthedocs picks up changes to our doc suite here and publishes automatically. So if you put a date of 12-18 on this people may think version 3.0.3 has already been released if they look at the docs.

So for now let's just put this as date of TBD; I'm also not sure if this warrants a bugfix release (3.0.3) or a minor release (3.1.0) - just depends on if you consider this a bug or a new feature

Reply:
Ok, I’ll remove the date shortly. Regarding the minor/patch, do you consider this more of adding new functionality, or filling in something that was missing?

Reply:
Did you want to push this to pypi today? I guess with our new build process there isn't that much downside to just releasing a 3.0.3 now